program_repair.py

```diff
@@ -23,5 +23,37 @@ def replace(input_string, line, column, length, new_string):
 
 
 def remove_leading_spaces(input_string):
-    # the only repair we can do now is remove leading spaces, more can be added!
     return '\n'.join([x.lstrip() for x in input_string.split('\n')])
+
+
+def remove_unexpected_char(input_string, line, column):
+    return delete(input_string, line, column, 1)
+
+
+# TODO: check if the user tried to print a variable
+def add_missing_quote(input_string, line, column, length):
```

Comment:
This is called add while the other function is called fix, can we make that the same?